ansible-service-broker.spec

```diff
@@ -191,12 +191,9 @@ rm -rf src
 install -d -p %{buildroot}%{_bindir}
 install -p -m 755 broker %{buildroot}%{_bindir}/asbd
 install -p -m 755 build/entrypoint.sh %{buildroot}%{_bindir}/entrypoint.sh
-install -p -m 755 build/%{name} %{buildroot}%{_bindir}/%{name}
 sed -i 's,/usr/local/%{name}/bin,/usr/libexec/%{name},g' %{buildroot}%{_bindir}/%{name}
-install -d -p %{buildroot}%{_docdir}/%{name}
 install -d -p %{buildroot}%{_sysconfdir}/%{name}
-install -p -m 755 etc/example-broker-config.yaml %{buildroot}%{_docdir}/%{name}/example-broker-config.yaml
-install -p -m 644 build/broker-config.yaml %{buildroot}%{_sysconfdir}/%{name}/broker-config.yaml
+install -p -m 644 etc/example-config.yaml %{buildroot}%{_sysconfdir}/%{name}/config.yaml
```

Comment:
@jmontleon do we want the example config file to be:
 etc/config.yaml.example in our source?

So a rename of that file is required?